StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/SimpleMandateContainerView.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  SimpleMandateContainerView.swift
+//  StripePaymentSheet
+//
+//  Created by Nick Porter on 10/3/24.
+//
+
+import Foundation
+import UIKit
+
+class SimpleMandateContainerView: UIView {
```

Comment:
I tried using a UILayoutGuide but could just not get it working... I think it's b/c the mandate is contain within a stack view.